packages/lesswrong/components/sequences/SequencesHoverOver.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from 'react';
+import { useMulti } from '../../lib/crud/withMulti';
+import { registerComponent, Components } from '../../lib/vulcan-lib';
+import Card from '@material-ui/core/Card';
+import { useSingle } from '../../lib/crud/withSingle';
+
+const styles = (theme: ThemeType): JssStyles => ({
+  root: {
+    padding: 16,
+    width: 450
+  },
+  title: {
+    ...theme.typography.body1,
+    ...theme.typography.postStyle,
+    fontVariant: "small-caps",
+  },
+  description: {
+    ...theme.typography.body2,
+    ...theme.typography.postStyle,
+    paddingTop: 8,
+    paddingBottom: 8,
+  },
+  author: {
+    color: theme.palette.text.dim
+  }
+});
+
+export const SequencesHoverOver = ({classes, sequence, showAuthor=true}: {
+  classes: ClassesType,
+  sequence: SequencesPageFragment|null,
+  showAuthor?: boolean
+}) => {
+  const { SequencesSmallPostLink, Loading, ContentStyles, ContentItemTruncated, UsersName } = Components
+
+  const { results: chapters, loading } = useMulti({
+    terms: {
+      view: "SequenceChapters",
+      sequenceId: sequence?._id,
+      limit: 100
+    },
+    collectionName: "Chapters",
+    fragmentName: 'ChaptersFragment',
+    enableTotal: false,
+  });
+  
+  return <Card className={classes.root}>
+    {!sequence && <Loading/>}
+    <div className={classes.title}>{sequence?.title}</div>
+    { showAuthor && sequence?.user &&
+      <div className={classes.author}>
+        by <UsersName user={sequence?.user} />
+      </div>}
+    <ContentStyles contentType="postHighlight" className={classes.description}>
+      <ContentItemTruncated
+        maxLengthWords={100}
+        graceWords={20}
+        rawWordCount={sequence?.contents?.wordCount || 0}
+        expanded={false}
+        getTruncatedSuffix={() => null}
+        dangerouslySetInnerHTML={{__html: sequence?.contents?.htmlHighlight || ""}}
+        description={`sequence ${sequence?._id}`}
+      />
+    </ContentStyles>
+    {!chapters && loading && <Loading />}
+    {chapters?.map((chapter) => <span key={chapter._id}>
+        {chapter.posts?.map(post => <SequencesSmallPostLink 
+                                      key={chapter._id + post._id} 
+                                      post={post}
+                                    />
+        )}
+      </span>
+    )}
```

Comment:
This section is incorrectly indented. An individual instance of incorrect indentation is not a huge deal, but this happens often enough that it seems worth flagging.

The most important rule is, if two things open and close something, they have to be on lines which are the same indent level. So the `</span>` line needs to be indented one less level to match the `{chapters?.map` line which has the `<span>`. There are some other indentation oddities stemming from that. I don't think we've written a conventions document that bans the indent-a-bunch-of-extra-levels-for-alignment thing, and there are codebases that do that, but it's not something we normally do in the LW codebase and I think it's usually bad.

The correctly indented version of this would be:

```
  {chapters?.map((chapter) => <span key={chapter._id}>
    {chapter.posts?.map(post =>
      <SequencesSmallPostLink 
        key={chapter._id + post._id} 
        post={post}
      />
    )}
  </span>)}
```